tests/fixups/TraceFixup/main.cpp

```diff
@@ -151,119 +156,149 @@ result_configuration configured_result(function_type type, function_result resul
+        TraceLoggingValue(callingmodule, "Caller"),
+        TraceLoggingInt64(TickStart.QuadPart, "Start"),
+        TraceLoggingInt64(TickEnd.QuadPart, "End")
+    ); // Field for your event in the form of (value, field name).
 }
 
-void Log_ETW_PostMsgW(const wchar_t * s)
+void Log_ETW_PostMsgW(const wchar_t* s)
 {
-	TraceLoggingWrite(g_Log_ETW_ComponentProvider, // handle to my provider
-		"TraceEvent",              // Event Name that should uniquely identify your event.
-		TraceLoggingValue(s, "Message")); // Field for your event in the form of (value, field name).
+    TraceLoggingWrite(g_Log_ETW_ComponentProvider, // handle to my provider
+        "TraceEvent",              // Event Name that should uniquely identify your event.
+        TraceLoggingValue(s, "Message")); // Field for your event in the form of (value, field name).
 
 }
 
 // Tear down the ETW Provider
 void Log_ETW_UnRegister()
 {
-	TraceLoggingUnregister(g_Log_ETW_ComponentProvider);
+    TraceLoggingUnregister(g_Log_ETW_ComponentProvider);
 }
 
 BOOL __stdcall DllMain(HINSTANCE, DWORD reason, LPVOID) noexcept try
 {
     if (reason == DLL_PROCESS_ATTACH)
     {
+        Log_ETW_Register();
+
+        std::wstringstream traceDataStream;
+
         if (auto config = PSFQueryCurrentDllConfig())
         {
             auto& configObj = config->as_object();
 
+            traceDataStream << " config:\n";
             if (auto method = configObj.try_get("traceMethod"))
             {
                 auto methodStr = method->as_string().string();
+                traceDataStream << " traceMethod:" << method->as_string().wide() << " ;";
+
                 if (methodStr == "printf"sv)
                 {
                     output_method = trace_method::printf;
-					Log("config traceMethod is printf");
+                    Log("config traceMethod is printf");
+                }
+                else if (methodStr == "eventlog"sv)
+                {
+                    output_method = trace_method::eventlog;
+                    Log("config traceMethod is eventlog");
                 }
-				else if (methodStr == "eventlog"sv)
-				{
-					output_method = trace_method::eventlog;
-					Log_ETW_Register();
-					Log("config traceMethod is eventlog");
-				}
-				else {
+                else {
                     // Otherwise, use the default (OutputDebugString)
-					Log("config traceMethod is default");
+                    Log("config traceMethod is default");
                 }
             }
 
             if (auto levels = configObj.try_get("traceLevels"))
             {
                 g_traceLevels = &levels->as_object();
+                traceDataStream << " traceLevels:\n";
 
                 // Set default level immediately for fallback purposes
                 if (auto defaultLevel = g_traceLevels->try_get("default"))
                 {
+                    traceDataStream << " default:\n" << defaultLevel->as_string().wide() << " ;";

```

Comment:
>\n [](start = 49, length = 2)

Is the new line intentional here? line 252 also has similar info, but does not have a new line.